src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -56,8 +58,15 @@ public Task ReceiveReminderAsync(string reminderName, byte[] state, TimeSpan due
     ///     A service fabric actor implementation that is responsible for creating and updating pull requests for dependency
     ///     updates.
     /// </summary>
-    public class PullRequestActor : IPullRequestActor, IRemindable, IActionTracker
+    public class PullRequestActor : IPullRequestActor, IRemindable, IActionTracker, IStatefulActor
```

Comment:
>IStatefulActor [](start = 84, length = 14)

NIT: the Stateful in this interface is redundant. All Actors are stateful by definition.

Reply:
What's a better name to explain what this type is... IDelegatedActor, IProxiedActor... I'm not 100% what the use case of the intercepty guy is to pick a good word here.

Reply:
The interceptor stuff is so we can log every actor method invocation to app insights, and so we can create a new scope and implementation type per method call. Unfortunately IActor is already used. I can't really think of any other name. Maybe IActorImplementation, thats kinda what the whole type is.

Reply:
Hrm... Uh... ITracedActor? Since it's used to enable the automatic tracing of stuff.

Reply:
Went with IActorImplementation.  I really wanted to just pass the whole actor, since it's the source of all 3 values, but that makes testing hard, since they don't have an actual actor, and I didn't want to introduce yet another interface IDelegatedActor, for them to construct and pass.